src/dal_alight/test.py

```
@@ -12,12 +14,13 @@ class AlightStory:
     HTML structure produced by ``AlightWidgetMixin``::
 
         <autocomplete-select>
-          <select slot="select" id="id_{field}">…</select>
-          <div slot="deck">
+          <input type="hidden" name="{field}" value="1"
+                 slot="values" data-label="Label">
+          <span slot="deck">
             <div data-value="1">Label<span class="clear">×</span></div>
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)